### ISSUE TITLE
Support I2C HID devices?

### ISSUE THREAD

**Issue by waitlamp** [OpenTabletDriver/OpenTabletDriver#1205]
Some laptop or 2-in-1 devices also equipped with digitizer
Tablet or graphic tablet normally use USB port
but these tablet PCs usually have I2C HID digitizer devices. 

(usually, they were detected as more than one devices in device manger)

Do you have any plans to adapt to I2C devices in the future?

thanks.

**Comment by adryzz** [comment]
Yes, but we need a way to access such I2C devices from user-level .NET/native C++ code

**Comment by InfinityGhost** [comment]
I2C HID will be supported, but not anytime soon.

**Comment by gonX** [comment]
Related to #1390 as its another request for a different backend

**Comment by Twilight-lxy** [comment]
I have a Lenovo Duet 2020 tablet, which is also a wocam I2C device. I want to use a stylus in Ubuntu, but Lenovo does not provide a driver for Ubuntu. I also hope to receive support through the OTD software.
